packages/esm-patient-attachments-app/src/attachments/camera-upload.component.tsx

```diff
@@ -53,7 +53,7 @@ const CameraUpload: React.FC<CameraUploadProps> = ({ onSavePhoto, onTakePhoto, c
     };
   }, []);
 
-  const willSaveImage = useCallback(
+  const willSaveImageOrPdf = useCallback(
```

Comment:
```suggestion
  const willSaveAttachment = useCallback(
```

Reply:
@ibacher thanks so much! Let me fix it right away!